src/hdf5/HDF5IO.cpp

```diff
@@ -31,7 +28,7 @@ HDF5IO::~HDF5IO()
 
 std::string HDF5IO::getFileName()
 {
-  return filename;
+  return this->m_filename;
```

Comment:
several cases in this file of using both `this->` and `m_` that I think we could simplify

Reply:
I've now replaced all occurrences of `this->m_` with just `m_`